web-external/src/style/full-viewport.styl

```diff
@@ -1,24 +1,23 @@
-
 html
-  height: 100%
+  height 100%
 
 body
-  position: absolute
-  top: 0
-  bottom: 0
-  left: 0
-  right: 0
+  position absolute
+  top 0
+  bottom 0
+  left 0
+  right 0
 
 #g-global-nav-container
-  z-index: 1
+  z-index 1
 
-#g-app-footer-container
-  position: fixed
-  bottom: 0
-  left: 0
-  right: 0
+DISABLE#g-app-footer-container
+  position fixed
+  bottom 0
+  left 0
+  right 0
 
-#g-app-body-container
-  height: 95%
-  padding-bottom: 88px
+DISABLE#g-app-body-container
+  height 95%
+  padding-bottom 88px
```

Comment:
What exactly is getting hidden?  Is there a way that we can get the footer bottom-aligned without this issue?  My CSS-foo is failing me.